src/store/modules/user/getters.ts

```diff
@@ -19,7 +19,8 @@ const getters: GetterTree <UserState, RootState> = {
         return state.currentFacility;
     },
     getInstanceUrl (state) {
-        return state.instanceUrl;
+        const baseUrl = process.env.VUE_APP_BASE_URL;
```

Comment:
Update api/index file as per the changes in the getter.

Reply:
Done sir.